aws-xray-recorder-sdk-benchmark/README.md

```diff
@@ -39,9 +40,9 @@ You should then start seeing the output of the benchmark:
 
 If you wish to run a specific benchmark, please run the following command:
 ```
-java -jar target/benchmarks.jar <Benchmark_name_here>
+java -jar aws-xray-recorder-sdk-benchmark-<VERSION>-jmh.jar <Benchmark_name_here>
```

Comment:
Though it means forwarding a property into the Gradle config like here

https://github.com/line/armeria/blob/master/benchmarks/build.gradle#L32

Reply:
I think that's only if we were using the `jmh` task with gradle, when executing the jar directly I verified that just passing the fully qualified name of a benchmark class or test works.